packages/mobile/src/navigator/DrawerTopBar.tsx

```diff
@@ -1,46 +1,62 @@
-import colorsV2 from '@celo/react-components/styles/colors.v2'
-import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import HamburgerIcon from '@celo/react-components/icons/Hamburger'
+import colors from '@celo/react-components/styles/colors.v2'
 import { useNavigation } from '@react-navigation/native'
 import * as React from 'react'
 import { StyleSheet, TouchableOpacity, View } from 'react-native'
-import Logo from 'src/icons/Logo.v2'
-import Hamburger from 'src/navigator/Hamburger'
+import Animated from 'react-native-reanimated'
 
-const DrawerTopBar = () => {
+interface Props {
+  middleElement?: React.ReactNode
+  scrollPosition?: Animated.Value<number>
+}
+
+// Test mocks would not work if import these together with Animated
+const { cond, greaterThan } = Animated
+
+function DrawerTopBar({ middleElement, scrollPosition }: Props) {
   const navigation = useNavigation()
+  const borderBottom = React.useMemo(
+    () => ({
+      borderBottomColor: colors.gray2,
+      borderBottomWidth: cond(greaterThan(scrollPosition ?? new Animated.Value(0), 0), 1, 0),
+    }),
+    [scrollPosition]
+  )
 
   return (
-    <View style={styles.container}>
+    <Animated.View style={[styles.container, borderBottom]}>
```

Comment:
When using `Animated.View`, it's more performant to pass a style that doesn't change.
We could easily change `useMemo` above to output the full style.